_rules/inline-link-in-paragraph-has-distinguishing-style-not-based-on-color-alone-88407d.md

```diff
@@ -0,0 +1,148 @@
+---
+id: 88407d
+name: Inline link has distinguishable style not based on color alone
+rule_type: atomic
+description: |
+  This rule checks that inline links have a style not based on color alone that distinguishes them from the surrounding text
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: The element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: The element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][].
+
+## Expectation
+
+Each target element has a [distinguishing style][] not based on color alone.
```

Comment:
It is not fully clear here (and in similar rules) whether the distinguishing style need to always be here or not. I probably wouldn't have blinked in normal circumstances, but given that some of the rules are only checking when the element is hovered/focused, the question pops here.

Which actually gets into another question: if the distinguishing style is not the same between the different states (e.g. underline normally, but border on hover/focus), is it good? It certainly should, but I don't fully manage to infer that from the expectation (the element doesn't have **a** (singular) distinguishing style, it has several and each of them is good).

Reply:
Hopefully, this is addressed in the revised version, with an atomic for non-hover, non-focus and another for hover, focus states.